src/main/java/com/waratek/spiracle/sql/util/SelectUtil.java

```diff
@@ -128,7 +128,6 @@ public static void executeQuery(String sql, ServletContext application, HttpServ
 					logger.error(conCloseException);
 				}
 			}
-			TagUtil.printPageFooter(out);
```

Comment:
My bad. Have reintroduced the method call to the 'finally' block and deleted the other.